bin/git-archive-file

```diff
@@ -1,27 +1,27 @@
 #!/usr/bin/env bash
 
 # extract current branch name
-BRANCH=$(git rev-parse --abbrev-ref HEAD 2> /dev/null)
+BRANCH=$(git rev-parse --abbrev-ref HEAD 2>/dev/null)
 
 # get name of the most top folder of current directory, used for the
 # output filename
-ARCHIVE_NAME=$(basename "$PWD")
+ARCHIVE_NAME=$(basename "$(git rev-parse --show-toplevel)")
 
 if [[ $BRANCH = tags* ]]; then
-    BRANCH=$(git describe)
-    echo Building for tag "\"$BRANCH\""
-    FILENAME=$ARCHIVE_NAME.$BRANCH.zip
+  BRANCH=$(git describe)
```

Comment:
Let's keep the original indentation to avoid generating unrelative diff.

Reply:
I will revert the indent changes. Looks like I might have forgotten to disable my auto-application of `shfmt`.